### PR TITLE
Task 16 Implement Home Screen

### PR DESCRIPTION
**Pull request by jsong336**
* Implemented basic UI components for Home Screen.
* Added go back link

<div>
  
  
  
  

</div>

### CODE REVIEW

**Comment by jhwan7** [comment]
maybe discuss the "go back" button with Erin.

**Comment by jsong336** [comment]
> maybe discuss the "go back" button with Erin.

yeah, linked her to reviewer

File: app/component/enter-room/EnterRoomBackground.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import {Image, ImageBackground, StyleSheet} from 'react-native'
+export default (props) => {
+    return (
+        <ImageBackground
+            style={[styles.backgroundImage, props.style]}
+            source={require('../../assets/enterscreen-background/enterscreen-background.png')}
```
Comment (jhwan7) [inline app/component/enter-room/EnterRoomBackground.tsx:7]: does this need to use require() ? To me import would make it look cleaner.

File: app/screen/EnterRoomScreen.tsx [hunk 2]
```diff
@@ -1,12 +1,117 @@
...
+import React, { useState } from 'react';
+import { View, Text, Pressable, StyleSheet, SafeAreaView } from 'react-native';
+import EnterRoomButton from '../component/enter-room/EnterRoomButton';
+import EnterRoomTextForm from '../component/enter-room/EnterRoomTextForm';
+import EnterRoomBackground from '../component/enter-room/EnterRoomBackground';
+import global, { COLORS } from '../styles/global';
+import GuessWhomLogo from '../component/GuessWhomLogo';
 
 export default () => {
+    const [shownForm, setShownForm] = useState<"Join" | "Create" | "">("");
```
Comment (jhwan7) [inline app/screen/EnterRoomScreen.tsx:10]: you might want to look into typescript interface

**Review by jhwan7 — APPROVED** [review]

File: app/component/enter-room/EnterRoomBackground.tsx (hunk 1 above)
Comment (jsong336) [reply app/component/enter-room/EnterRoomBackground.tsx:7]: Using import was complaining in linter. Is there way to fix it in typescript?

**Comment by erin4103** [comment]
Updated UI prototype with back button : https://xd.adobe.com/view/edd32928-afb3-46c3-9e93-998c5fe50882-b880/screen/34657b96-6add-4b37-88c2-8814720e41b8/specs/

File: app/screen/EnterRoomScreen.tsx (hunk 2 above)
Comment (jsong336) [reply app/screen/EnterRoomScreen.tsx]: Added enum

**Pull request merged**